### ISSUE TITLE
Allow "edit in ScalaFiddle" for any snippet

### ISSUE THREAD

**Issue by ktoso** [lightbend/paradox#124]
Sometimes when a snippet is interesting, but not complete, and I may want to hack on it,
i may want to "get this snippet to scalafiddle and let me edit it" - this could open in a new window.

**Comment by andreaTP** [comment]
Actually this is a feature request to be moved to scalafiddle itself.
but great @ochrons already provided it:
http://doc.akka.io/docs/akka/current/scala/actors.html#here-is-another-example-that-you-can-edit-and-run-in-the-browser-

:-)

**Comment by ktoso** [comment]
I don't think so - it's about making a snippet be reloaded as fiddle as I may want to edit it to play around. I don't think we'd want to load all snippets on a page as fiddles initially - it would not make too much sense and take resources for snippets that are not even intended to be run unless edited

**Comment by hepin1989** [comment]
Do we support scastie ? like https://scastie.scala-lang.org/sadhen/vjSdQp9XQ5uFu23U7nTF0w ,
@andreaTP

**Comment by andreaTP** [comment]
given version `0.4` of paradox this is now feasible (IMO) e.g. having all directives as fiddle (since it degrades gracefully now).

But this decision is above me.

**Comment by andreaTP** [comment]
ok, that's different from what I understood @ktoso and will require further investigation; in my opinion there are a couple of technical limitations that make this feature mostly unpractical (remember that we are still running in JS and e.g. Akka documentation is pretty thigh to JVM, IO, Files etc. etc.)

BTW @hepin1989 I completely miss your comment, really sorry, as for now only scalafiddle is supported.
